intro-HPC/ch_troubleshooting.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{Troubleshooting}
+\label{ch:troubleshooting}
+
+
+
+\section{Walltime issues}
+If you get from your job output error a message similar to this:
+
+\begin{prompt}
+ =>> PBS: job killed: walltime 146 exceeded limit 120
+\end{prompt}
+
+You job has exceeded the requested walltime. Probably the job requires more time to be executed.
+See section~\ref{sec:specifying-walltime-requirements} for more information about how to request the walltime in an accurate way.
```

Comment:
I would drop the `in an accurate way` (is there an 'inaccurate' way of requesting walltime)?